README.md

```diff
@@ -214,6 +214,15 @@ This will:
 * Run `crank` again with the state already downloaded, to avoid any unintended diffs
 * Use `dyff` to generate the diffs between both results
 
+### Generate diff against live version with debugger
+
+If you want to print the diff while using the debugger you can simply do this:
+
+* Star the local debugging session as described
```

Comment:
Should be start :D

Reply:
Ah... I thought the two typos were on the same line :D